internal/users/edit.go

```diff
@@ -14,34 +15,17 @@ import (
 	"golang.org/x/crypto/bcrypt"
 )
 
-type EditUserRequest struct {
-	ID        int    `json:"-"`
-	Name      string `json:"name" validate:"omitempty,checkName"`
-	Password  string `json:"password" validate:"omitempty,checkPassword"`
-	Email     string `json:"email" validate:"omitempty,email"`
-	BirthDate string `json:"birthdate" validate:"omitempty,checkDate"`
-	Nickname  string `json:"nickname" validate:"omitempty,checkNickname"`
-	Bio       string `json:"bio" validate:"omitempty,checkBio"`
-	Location  string `json:"location" validate:"omitempty,checkLocation"`
-}
+type editUserRequest createUserRequest
```

Comment:
these are not the same, most (if not all) of the fields that are required in the create request are optional in the edit request

Reply:
and this should be covered by tests too